pkcs12/src/lib.rs

```diff
@@ -14,4 +14,16 @@
     unused_qualifications
 )]
 
-//! TODO: PKCS#12 crate
+//! TODO: complete PKCS#12 crate
+
+use cfg_if::cfg_if;
+
+cfg_if! {
+    if #[cfg(feature = "alloc")] {
+
+        #[macro_use]
+        extern crate alloc;
+
+        pub mod kdf;
+    }
+}
```

Comment:
Per my comment about a hard dependency on `alloc` above:

```suggestion
extern crate alloc;

pub mod kdf;
```